### PR TITLE
Bottom sheet

### PR DESCRIPTION
**Pull request by tereshkevich1**

### CODE REVIEW

File: app/src/main/java/com/example/banking/accounts_screen/AccountsViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import com.example.banking.models.CardState
+import com.example.banking.models.Transaction
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import java.util.Date
+
+class AccountsViewModel : ViewModel() {
+    private val _account = MutableStateFlow(Account("name", "1241244132525", "134234235", true))
+    val account: StateFlow<Account> get() = _account
+
```
Comment (Razor0744) [inline app/src/main/java/com/example/banking/accounts_screen/AccountsViewModel.kt:14]: val account: StateFlow<Account> = _account.asStateFlow

File: app/src/main/java/com/example/banking/accounts_screen/AccountsViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            "Google",
+            Date(),
+            1000,
+            CardState.IN_PROGRESS
+        ),
+        Transaction("Google", Date(), 1000, CardState.EXECUTED)
+    )
+    fun loadAccount() {
+        _account.value = Account("name", "1241244132525", "134234235", true)
+    }
```
Comment (Razor0744) [inline app/src/main/java/com/example/banking/accounts_screen/AccountsViewModel.kt:34]: use _account.emit() with the coroutine if you can

File: app/src/main/java/com/example/banking/accounts_screen/bottomSheet/AccountsBottomSheet.kt [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+fun AccountsBottomSheet(
+    accountsList: List<Account>,
+    onDismiss: () -> Unit,
+    onAccountClick: (Account) -> Unit,
+    account: Account
+) {
+    val modalBottomSheetState = rememberModalBottomSheetState()
+    val containerColor = colorResource(id = R.color.surface_background_color)
+    ModalBottomSheet(
+        onDismissRequest = onDismiss, sheetState = modalBottomSheetState,
```
Comment (Razor0744) [inline app/src/main/java/com/example/banking/accounts_screen/bottomSheet/AccountsBottomSheet.kt:23]: space after variable

File: app/src/main/java/com/example/banking/transaction_screen/TransactionScreen.kt [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+import androidx.compose.ui.tooling.preview.Preview
+import com.example.banking.R
+import com.example.banking.ui.theme.BankingTheme
+
+@Composable
+fun TransactionScreen(){
+    Column {
+        
+    }
+
```
Comment (Razor0744) [inline app/src/main/java/com/example/banking/transaction_screen/TransactionScreen.kt:18]: remove empty fun

File: app/src/main/java/com/example/banking/accounts_screen/AccountsViewModel.kt (hunk 1 above)
Comment (tereshkevich1) [reply app/src/main/java/com/example/banking/accounts_screen/AccountsViewModel.kt:14]: fixed

File: app/src/main/java/com/example/banking/accounts_screen/AccountsViewModel.kt (hunk 2 above)
Comment (tereshkevich1) [reply app/src/main/java/com/example/banking/accounts_screen/AccountsViewModel.kt:34]: fixed

File: app/src/main/java/com/example/banking/accounts_screen/bottomSheet/AccountsBottomSheet.kt (hunk 3 above)
Comment (tereshkevich1) [reply app/src/main/java/com/example/banking/accounts_screen/bottomSheet/AccountsBottomSheet.kt:23]: fixed

File: app/src/main/java/com/example/banking/transaction_screen/TransactionScreen.kt (hunk 4 above)
Comment (tereshkevich1) [reply app/src/main/java/com/example/banking/transaction_screen/TransactionScreen.kt:18]: fixed

**Pull request merged**